### ISSUE TITLE
Unable to build cardano-graphql using docker

### ISSUE THREAD

**Issue by PegasusPool** [input-output-hk/cardano-graphql#443]
#### Platform

- [X] Linux / Ubuntu
- [ ] Linux / Other
- [ ] macOS
- [ ] Windows

**Platform version**: 

#### Runtime

- [ ] Node.js
- [X] Docker

**Runtime version**: 
```console
docker version
Client:
 Version:           19.03.8
 API version:       1.40
 Go version:        go1.13.8
 Git commit:        afacb8b7f0
 Built:             Fri Dec 18 12:15:19 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          19.03.8
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.8
  Git commit:       afacb8b7f0
  Built:            Fri Dec  4 23:02:49 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.3.3-0ubuntu2.2
  GitCommit:        
 runc:
  Version:          spec: 1.0.1-dev
  GitCommit:        
 docker-init:
  Version:          0.18.0
  GitCommit:        

```

### Steps to reproduce the bug

- Checkout to 3.2.0 tag
- Execute `docker-compose up -d --build && docker-compose logs -f`

### What is the expected behavior?

Cardano-graphql is built and running successfully

### Actual behavior

```
...
Step 23/80 : COPY /cardano-node /app/src/cardano-node
 ---> Using cache
 ---> ddafab69ae1d
Step 24/80 : WORKDIR /app/src/cardano-node
 ---> Using cache
 ---> 21337519b4ae
Step 25/80 : RUN cabal install cardano-cli   --install-method=copy   --installdir=/usr/local/bin   -f -systemd
 ---> Running in e0de7910376d
Resolving dependencies...
cabal: Could not resolve dependencies:
[__0] unknown package: cardano-cli (user goal)
[__0] fail (backjumping, conflict set: cardano-cli)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: cardano-cli

ERROR: Service 'cardano-graphql' failed to build: The command '/bin/sh -c cabal install cardano-cli   --install-method=copy   --installdir=/usr/local/bin   -f -systemd' returned a non-zero code: 1
```

#### Note

The previous version (3.1.1) was built and ran successfully on this machine.
I ran `docker-compose down -v` before trying to update to 3.2.0.

**Comment by rhyslbw** [comment]
The repo has `cardano-node` as a git submodule.

https://github.com/input-output-hk/cardano-graphql#getting-started

**Comment by wrmarchetto** [comment]
It appears the cardano-node project was not included in the 3.2.2 release tarball (the folder is empty).

**Comment by tanmaster** [comment]
> 
> 
> It appears the cardano-node project was not included in the 3.2.2 release tarball (the folder is empty).

You most likely skipped this step:

`git clone \
  --single-branch \
  --branch <VERSION> \
  --recurse-submodules \
  https://github.com/input-output-hk/cardano-graphql.git \
  && cd cardano-graphql`
